docs/source/architecture.rst

```diff
@@ -1,10 +1,19 @@
 
-************
-Architecture
-************
+******************
+Architecture Guide
+******************
 
 The following diagram shows a high-level view of the Sawtooth architecture.
 
+This guide describes the design and architecture of the Hyperledger Sawtooth,
```

Comment:
Should "the" in "the Hyperledger Sawtooth" be removed, or is it supposed to say "the Hyperledger Sawtooth platform"?

Reply:
Probably should just remove the "the"

Reply:
Thanks, Adam and Peter!  I just pushed a fixup commit that removes the "the".